### PR TITLE
Update SSH_ROOT_PASSWORD in example.conf files

### PR DESCRIPTION
**Pull request by jsmeix**
In example.conf files use
```
{ SSH_ROOT_PASSWORD='...' ; } 2>>/dev/$SECRET_OUTPUT_DEV
```
(i.e. with SECRET_OUTPUT_DEV)

### CODE REVIEW

**Comment by jsmeix** [comment]
@rear/contributors 
unless there are objections
I would like to merge it tomorrow afternoon.

**Review by schlomo — APPROVED** [review]

File: usr/share/rear/conf/examples/SLE11-ext3-example.conf [hunk 1]
```diff
@@ -19,9 +19,10 @@ BACKUP_URL=nfs://your.NFS.server.IP/path/to/your/rear/backup
 # Keep an older copy of the backup in a HOSTNAME.old directory
 # provided there is no '.lockfile' in the HOSTNAME directory:
 NETFS_KEEP_OLD_BACKUP_COPY=yes
-# This option defines a root password to allow SSH connection
-# whithout a public/private key pair
-#SSH_ROOT_PASSWORD='password_on_the_rear_recovery_system'
+# Set a root password for the ReaR recovery system in a confidential way
+# to allow SSH connection whithout a public/private key pair.
```
Comment (pcahyna) [inline usr/share/rear/conf/examples/SLE11-ext3-example.conf:23]: without

File: usr/share/rear/conf/examples/SLE12-SP2-btrfs-example.conf [hunk 2]
```diff
@@ -65,9 +65,10 @@ BACKUP_PROG_INCLUDE=( /var/cache /var/lib/mailman /var/tmp /var/lib/pgsql /usr/l
 # The following POST_RECOVERY_SCRIPT implements during "rear recover"
 # btrfs quota setup for snapper if that is used in the original system:
 POST_RECOVERY_SCRIPT=( 'if snapper --no-dbus -r $TARGET_FS_ROOT get-config | grep -q "^QGROUP.*[0-9]/[0-9]" ; then snapper --no-dbus -r $TARGET_FS_ROOT set-config QGROUP= ; snapper --no-dbus -r $TARGET_FS_ROOT setup-quota && echo snapper setup-quota done || echo snapper setup-quota failed ; else echo snapper setup-quota not used ; fi' )
-# This option defines a root password to allow SSH connection
-# whithout a public/private key pair
-#SSH_ROOT_PASSWORD='password_on_the_rear_recovery_system'
+# Set a root password for the ReaR recovery system in a confidential way
+# to allow SSH connection whithout a public/private key pair.
```
Comment (pcahyna) [inline usr/share/rear/conf/examples/SLE12-SP2-btrfs-example.conf:69]: without

File: usr/share/rear/conf/examples/SLE12-btrfs-example.conf [hunk 3]
```diff
@@ -38,9 +38,10 @@ NETFS_KEEP_OLD_BACKUP_COPY=yes
 #   findmnt -n -r -o TARGET -t btrfs | grep -v '^/$' | egrep -v 'snapshots|crash'
 # to generate the values:
 BACKUP_PROG_INCLUDE=( /home /var/tmp /var/spool /var/opt /var/log /var/lib/pgsql /var/lib/mailman /var/lib/named /usr/local /tmp /srv /boot/grub2/x86_64-efi /opt /boot/grub2/i386-pc )
-# This option defines a root password to allow SSH connection
-# whithout a public/private key pair
-#SSH_ROOT_PASSWORD='password_on_the_rear_recovery_system'
+# Set a root password for the ReaR recovery system in a confidential way
+# to allow SSH connection whithout a public/private key pair.
```
Comment (pcahyna) [inline usr/share/rear/conf/examples/SLE12-btrfs-example.conf:42]: without

File: usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf [hunk 4]
```diff
@@ -37,9 +37,10 @@ NETFS_KEEP_OLD_BACKUP_COPY=yes
 # Make sure to exclude all SAP HANA related volume groups and mount points - modify next line with your HANA mount points:
 EXCLUDE_VG=( vgHANA-data-HC2 vgHANA-data-HC3 vgHANA-log-HC2 vgHANA-log-HC3 vgHANA-shared-HC2 vgHANA-shared-HC3 )
 BACKUP_PROG_EXCLUDE+=( '/hana/*' )
-# This option defines a root password to allow SSH connection
-# whithout a public/private key pair
-#SSH_ROOT_PASSWORD='password_on_the_rear_recovery_system'
+# Set a root password for the ReaR recovery system in a confidential way
+# to allow SSH connection whithout a public/private key pair.
```
Comment (pcahyna) [inline usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf:41]: without

File: usr/share/rear/conf/examples/SLE11-ext3-example.conf (hunk 1 above)
Comment (jsmeix) [reply usr/share/rear/conf/examples/SLE11-ext3-example.conf]: fixed

File: usr/share/rear/conf/examples/SLE12-SP1-btrfs-example.conf [hunk 5]
```diff
@@ -60,9 +60,10 @@ COPY_AS_IS+=( /usr/lib/snapper/installation-helper /etc/snapper/config-templates
 #   findmnt -n -r -o TARGET -t btrfs | grep -v '^/$' | egrep -v 'snapshots|crash'
 # to generate the values:
 BACKUP_PROG_INCLUDE=( /var/tmp /srv /var/lib/pgsql /var/spool /var/lib/libvirt/images /var/opt/ /tmp /var/lib/named /var/log /boot/grub2/i386 /var/lib/mariadb /home /var/lib/mailman /opt /usr/local /boot/grub2/x86_64 )
-# This option defines a root password to allow SSH connection
-# whithout a public/private key pair
-#SSH_ROOT_PASSWORD='password_on_the_rear_recovery_system'
+# Set a root password for the ReaR recovery system in a confidential way
+# to allow SSH connection whithout a public/private key pair.
```
[the remark this answers is not in the document]
Comment (jsmeix) [reply usr/share/rear/conf/examples/SLE12-SP1-btrfs-example.conf]: fixed

File: usr/share/rear/conf/examples/SLE12-SP2-btrfs-example.conf (hunk 2 above)
Comment (jsmeix) [reply usr/share/rear/conf/examples/SLE12-SP2-btrfs-example.conf]: fixed

File: usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf (hunk 4 above)
Comment (jsmeix) [reply usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf]: fixed

File: usr/share/rear/conf/examples/SLE12-btrfs-example.conf (hunk 3 above)
Comment (jsmeix) [reply usr/share/rear/conf/examples/SLE12-btrfs-example.conf]: fixed

**Comment by jsmeix** [comment]
@pcahyna 
thank you for your sharp-sighted review!

**Review by pcahyna — APPROVED** [review]
@jsmeix you are welcome! I suggest to squash your new commits with the respective original commits to reduce the number of commits in the history before merging.

**Comment by jsmeix** [comment]
@pcahyna 
ah - yes - thank you for the hint about squashing commits.

I like to be able to see the history how things were developed
(and the overall result is visible in the merge commit)
so normally I don't like squashing commits so that
I am not used to think about squashing commits.

But in this case it would be better to squash them.

**Comment by jsmeix** [comment]
This was my very first try to use "Squash and merge"
in the GitHub web frontend - let's see what the result is...

**Comment by jsmeix** [comment]
Ah!
https://github.com/rear/rear/commit/7c5c9bc83db5202fb515cbaa803efcd18d9fcd1e
shows me that I also have to manually squash the commit messages.
Next time I (hopefully) remember that and do better.

**Comment by pcahyna** [comment]
> ...ah! [7c5c9bc](https://github.com/rear/rear/commit/7c5c9bc83db5202fb515cbaa803efcd18d9fcd1e) shows me that I also have to manually squash the merge commit message. Next time I (hopefully) remember that and do better.

@jsmeix I usually squash only commits that I need manually using `git rebase --interactive --autosquash` and I force push. This allows me to squash only commits that I want to have squashed and adjust the commit messages. For example, in this case, I would have squashed both commits Update SLE11-SLE12-SAP-HANA-UEFI-example.conf together, both Update SLE12-SP1-btrfs-example.conf commits together etc. but maybe not all commits together.

**Pull request merged**